shapely/tests/test_plotting.py

```diff
@@ -1,18 +1,41 @@
 import pytest
 from numpy.testing import assert_allclose
 
-import shapely
-from shapely import box, LineString, MultiLineString, Point
-from shapely.plotting import plot_line, plot_points, plot_polygon
+from shapely import box, get_coordinates, LineString, MultiLineString, Point
+from shapely.plotting import patch_from_polygon, plot_line, plot_points, plot_polygon
 
 pytest.importorskip("matplotlib")
```

Comment:
I see... it's because I moved the imports to the top of `plotting.py` so even when matplotlib wasn't installed before, `pytest` would skip here.

Reply:
I prefer to see this kept to skip, since matplotlib is not a requirement.

Reply:
Sounds good. Do you want me to revert what I did in the GitHub actions config then? Because there are some jobs that straight up did not install matplotlib prior to this PR

Reply:
yes, probably those too, thanks!

Reply:
>  Because there are some jobs that straight up did not install matplotlib prior to this PR

That was intentional: since matplotlib is optional, we should have some builds without it, to ensure shapely properly works without it.